api/v1alpha1/clusterorder_types.go

```diff
@@ -63,20 +66,31 @@ const (
 	// ClusterOrderConditionControlPlaneAvailable means the cluster control plane is ready
 	ClusterOrderConditionControlPlaneAvailable ClusterOrderConditionType = "ControlPlaneAvailable"
 
-	// ClusterOrderConditionNodePoolAvailable means the node pool has the correct number of nodes
-	ClusterOrderConditionNodePoolAvailable ClusterOrderConditionType = "NodePoolAvailable"
-
 	// ClusterOrderConditionAvailable means the cluster is available
 	ClusterOrderConditionAvailable ClusterOrderConditionType = "Available"
 )
 
+// ClusterOrderClusterReferenceType contains a reference to the namespace created by this ClusterOrder
+type ClusterOrderClusterReferenceType struct {
+	// Namespace that contains the HostedCluster resource
+	ClusterNamespace   string `json:"clusterNamespace"`
+	ClusterName        string `json:"clusterName"`
```

Comment:
How about `HostedClusterName` here since it's specific to that particular Kind? That would match the field below also.